database/user.go

```diff
@@ -349,6 +349,7 @@ func (db *DB) UserCreate(ctx context.Context, emailAddr, pass, sub string, tier
 		Sub:                              sub,
 		Tier:                             tier,
 	}
+	// TODO This part can race and create multiple accounts with the same email, unless we add DB-level uniqueness restriction.
```

Comment:
kind of a scary TODO to leave in here like that, can't we quickly add a unique index instead?

Reply:
We can. We removed it when we added the challenge/response because users wanted the email to be optional and the challenge/response doesn't provide an email. So, accounts registered with that will have empty emails, which won't be possible if they are unique